pymc/sampling/mcmc.py

```diff
@@ -602,7 +606,6 @@ def sample(
         f"({n_tune*n_chains:_d} + {n_draws*n_chains:_d} draws total) "
         f"took {t_sampling:.0f} seconds."
     )
-    mtrace.report._log_summary()
```

Comment:
Inbetween the line 574 `mtrace = MultiTrace(traces)[:length]` where the `MultiTrace` was created, no warnings were added to `mtrace`.
Therefore, there are no warnings to log and the `_log_summary()` call can safely be removed.